app/src/main/java/com/duckduckgo/app/brokensite/view/SiteProtectionsToggle.kt

```diff
@@ -80,4 +82,11 @@ class SiteProtectionsToggle @JvmOverloads constructor(
             }
         }
     }
+
+    private fun areProtectionsEnabled(): Boolean = when (state) {
+        SiteProtectionsState.ENABLED -> true
+        SiteProtectionsState.DISABLED,
+        SiteProtectionsState.DISABLED_BY_REMOTE_CONFIG,
+        -> false
+    }
```

Comment:
This can be simplified (unless there is a reason for the when):
```
private fun areProtectionsEnabled(): Boolean {
    return state == SiteProtectionsState.ENABLED
}
```

Reply:
The only reason I chose to use when here (and in general with enums / sealed classes) is the extra safety in case e.g. we add more states later -> With `when`, a developer will be forced by the compiler to edit this method, with `if`, the logic here may silently become incorrect.